offchain/danaswap-ui/flake-module.nix

```diff
@@ -6,25 +6,43 @@
       inherit (config) cat-lib dream2nix;
       ui =
         (dream2nix.lib.makeOutputs { source = ./.; settings = [{ subsystemInfo.nodejs = 16; }]; }).packages.danaswap-ui;
+
+      fa_sprite_util = pkgs.callPackage ./tools/fa_sprite_util/default.nix { };
```

Comment:
Is there a reason to use snake_case? I'm used to camelCase in nix-code for attributes/varibales or in cases where packages are named to kebab-case